pyhf/commandline.py

```diff
@@ -25,3 +27,18 @@ def xml2json(entrypoint_xml, basedir, output_file, track_progress):
     else:
         json.dump(spec, open(output_file, 'w+'), indent=4, sort_keys=True)
         log.info("Written to {0:s}".format(output_file))
+
+@pyhf.command()
+@click.argument('workspace', default = '-')
+@click.option('--output-file', help='The location of the output json file. If not specified, prints to screen.', default=None)
+def cls(workspace,output_file):
+    specstream = click.open_file(workspace)
```

Comment:
not sure how to check for existence in case of non `-` in a click-y way

Reply:
Linux-y would also allow someone to pipe-in the JSON output into this function, no? `pyhf xml2json .... | pyhf cls`

Reply:
this should already work. my point is that for it I had to remove the `type = click.Path`

Reply:
the reason it works is that click is smart about `-` so wherever it is '-' the stdin (if mode 'r') or stdout (if mode 'w') is written to